src/controller/CommissioningDelegate.h

```diff
@@ -22,9 +22,7 @@
 #include <app/ClusterStateCache.h>
 #include <app/OperationalSessionSetup.h>
 #include <controller/CommissioneeDeviceProxy.h>
-#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
 #include <controller/jcm/TrustVerification.h> // nogncheck
```

Comment:
If this is no longer conditional:
```suggestion
#include <controller/jcm/TrustVerification.h>
```

Reply:
My point is: if it's not conditional, remove the nogncheck.  If it becomes conditional again, re-add it then.